### DISCUSSION TITLE
Creating MultiPolygon json for a State.

### DISCUSSION THREAD

**Discussion by parthasai** [Sunbird-cQube/community#40] [Q&A] [answered]
Hi is there an easy way to prepare the multipolygon json file for any state for representation on map ?

**Comment by Sharathnaik38** [comment]
Please follow below steps,
1. Refer this path for creating json file of state coordinates. (https://github.com/Sunbird-cQube/cQube_Workflow/tree/main/development/ui/dashboard_ui/maps)
2. Replicate the same json with uniqe state code and  coordinates

**Reply by parthasai** [reply]
Thanks for the inputs.

Are there any open apis where we can get the state border coordinates ? 

Or will we have to fetch each coordinates manually from Google Maps or other sources?

**Comment by Sharathnaik38** [comment, marked as answer]
There is no particular apis, we need to google search  the coordinates(geojson data) for respective states.
